src/client/store/reducers/audioReducer.js

```diff
@@ -12,16 +12,13 @@ import {
 
 const audioContext = Pizzicato.context;
 const analyser = audioContext.createAnalyser();
-analyser.fftSize = 128;
 
-const initialState = {
+export const initialState = {
```

Comment:
No need to export initial state for testing proposals, you can create there you own

Reply:
you told that it could be such, rewrited